### ISSUE TITLE
Error when starting the application

### ISSUE THREAD

**Issue by kittaakos** [theia-ide/theia#1555]
The WS root is the `theia` Git repository itself:

```
16:03:15.152 abstract-resource-preference-provider.ts:6 Uncaught (in promise) A resource provider for 'file:///Users/akos.kitta/git/theia/.theia/settings.json' is not registered.
rejected @ abstract-resource-preference-provider.ts:6
Promise.then (async)
step @ abstract-resource-preference-provider.ts:6
fulfilled @ abstract-resource-preference-provider.ts:6
Promise.then (async)
step @ abstract-resource-preference-provider.ts:6
(anonymous) @ abstract-resource-preference-provider.ts:6
__awaiter @ abstract-resource-preference-provider.ts:6
AbstractResourcePreferenceProvider.init @ abstract-resource-preference-provider.ts:31
_postConstruct @ instantiation.js:27
resolveInstance @ instantiation.js:47
(anonymous) @ resolver.js:72
resolve @ resolver.js:96
(anonymous) @ container.js:245
Container._get @ container.js:236
Container.get @ container.js:195
(anonymous) @ preference-frontend-module.ts:19
get @ preference-service.ts:55
(anonymous) @ preference-service.ts:70
get @ preference-service.ts:68
PreferenceServiceImpl.onStart @ preference-service.ts:62
(anonymous) @ frontend-application.ts:198
step @ frontend-application.ts:6
(anonymous) @ frontend-application.ts:6
(anonymous) @ frontend-application.ts:6
__awaiter @ frontend-application.ts:6
FrontendApplication.startContributions @ frontend-application.ts:176
(anonymous) @ frontend-application.ts:90
step @ frontend-application.ts:6
(anonymous) @ frontend-application.ts:6
(anonymous) @ frontend-application.ts:6
__awaiter @ frontend-application.ts:6
FrontendApplication.start @ frontend-application.ts:88
start @ index.js:23
Promise.then (async)
Object.defineProperty.value @ index.js:55
__webpack_require__ @ bootstrap e2bb08d6cf13f8686a65:659
Object.defineProperty.value @ bootstrap e2bb08d6cf13f8686a65:708
(anonymous) @ bootstrap e2bb08d6cf13f8686a65:708

```

**Comment by elaihau** [comment]
I see the same error in my local environment. 

Here is how to reproduce it:
1) have a theia instance running on my localhost:3000 `~/dev/theia/examples/browser$ yarn run start`
2) rebuild the product `~/dev/theia$ yarn`
3) stop the theia instance started in step 1 by ctrl-c, and rerun `~/dev/theia/examples/browser$ yarn run start`
4) see the same errors that kittaakos reported

The errors do not show up again if I reload theia from the browser.

**Comment by kittaakos** [comment]
I will check what's the problem with this.

**Comment by kittaakos** [comment]
I will not have time to work on this now. Please grab the task if you want.

**Comment by vince-fugnitto** [comment]
Closed as part of #2460
